### ISSUE TITLE
TX Everywhere

### ISSUE THREAD

**Issue by Ap0ph1s69** [egzumer/uv-k5-firmware-custom#486]
any way to enable
Enable TX everywhere except Air Band
just for the hell of it would love to see that implemented .

**Comment by ErikS-web** [comment]
Yep already present and [documented in the Online manual (wiki)](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation#how-to-unlock-tx-on-all-bands)
